build-dmg-ci.sh

```
@@ -0,0 +1,366 @@
+  local p12_file
+  p12_file="$(mktemp /tmp/affine-cert.XXXXXX.p12)"
+
+  log "Creating temporary keychain"
+  security create-keychain -p "$KEYCHAIN_PASSWORD" "$TEMP_KEYCHAIN_NAME"
+  security set-keychain-settings -lut 21600 "$TEMP_KEYCHAIN_NAME"
+  security unlock-keychain -p "$KEYCHAIN_PASSWORD" "$TEMP_KEYCHAIN_NAME"
+
+  log "Adding temporary keychain to search list"
+  local current_keychains
+  current_keychains="$(security list-keychains -d user | tr -d '"')"
+  security list-keychains -d user -s "$keychain_path" $current_keychains
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)